Snowflake.Data/Logger/EasyLoggerUtil.cs

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using Mono.Unix;
+
+namespace Snowflake.Data.Log
+{
+    internal class EasyLoggerUtil
```

Comment:
Now the code in this class has no more connection to the Easy Logging feature. It could be renamed to UnixFilePermissionsUtil or UnixFilePermissionsHelper or ... I would like UnixFilePermissionsConverter best.

Reply:
Renamed to `UnixFilePermissionsConverter`

Reply:
Fixed